chain-impl-mockchain/doc/format.md

```diff
@@ -149,8 +149,8 @@ With the following serialization:
   * Type=2 utxo witness scheme (64 bytes):
     * ED25519 Signature (64 bytes)
   * Type=3 Account witness (68 bytes):
-    * Account Counter (4 bytes : TODO-ENDIANNESS)
     * ED25519 Signature (64 bytes)
+    * Account Counter (4 bytes in big endian)
```

Comment:
hopefully nobody minds that I moved this around (no particular reason to swap the order. I just happen to code it this way)